### PR TITLE
DWPF-1685 Improve page bookmarks

### PR DESCRIPTION
**Pull request by SamDudley**
- remove bookmark buttons added to bookmark list
- new icons and animations
- bookmark card for the new homepage
- improved accessibility
- per user bookmark index page which isn't exposed yet

### CODE REVIEW

File: src/core/templates/1_col_content.html [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {% extends "base.html" %}
+{% load bookmarks %}
 
 {% block main_content %}
     {% block pre_content %}{% endblock %}
     <div class="ws-primary-content">
         <article>
+            </div>
```
Comment (marcelkornblum) [inline src/core/templates/1_col_content.html:8]: is this a mistake? looks wonky :)

File: src/dw_design_system/dwds/elements/icon.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.icon {
+    // FIXME: dicuss with team
```
Comment (marcelkornblum) [inline src/dw_design_system/dwds/elements/icon.scss:2]: let's have this discussion :)

File: src/dw_design_system/dwds/styles/animations.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@keyframes throb {
```
Comment (marcelkornblum) [inline src/dw_design_system/dwds/styles/animations.scss:1]: so bouncy!

File: src/dw_design_system/dwds/styles/temporary-base.scss [hunk 4]
```diff
@@ -48,10 +48,6 @@ body {
 	h3 {
 		font-size: var(--s1);
 	}
-
-	a {
-		color: var(--color-link);
-	}
```
Comment (marcelkornblum) [inline src/dw_design_system/dwds/styles/temporary-base.scss:54]: why remove this?

File: src/dw_design_system/dwds/styles/typography.scss [hunk 5]
```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto+Flex:opsz,wght@8..144,100..1000&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```
Comment (marcelkornblum) [inline src/dw_design_system/dwds/styles/typography.scss:1]: still want to inspect this - i would like to hear about the issue but i also think this feature is a funny reason to be losing the font. can we chat?

File: src/interactions/templates/interactions/bookmark_card.html [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{% extends 'dwds/elements/extendable/card.html' %}
+{% load bookmarks %}
+{% block card_content %}
+<h3>Your bookmarks</h3>
+{% bookmark_list user limit=5 %}
```
Comment (marcelkornblum) [inline src/interactions/templates/interactions/bookmark_card.html:5]: no `limit` for now I tihnk

File: src/interactions/templates/interactions/bookmark_list.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
+{% load static wagtailcore_tags %}
+
+<style>
```
Comment (marcelkornblum) [inline src/interactions/templates/interactions/bookmark_list.html:3]: this ought to be in a SCSS file IMO

File: src/interactions/templates/interactions/bookmark_index.html [hunk 8]
```diff
@@ -0,0 +1,12 @@
+{% extends 'dwds_content.html' %}
+
+{% load bookmarks %}
+
+{% block title %}{{ user.profile.full_name }}'s bookmarks{% endblock %}
+
+{% block primary_content %}
+<h1 class="govuk-heading-l">{{ user.profile.full_name }}'s bookmarks</h1>
```
Comment (marcelkornblum) [inline src/interactions/templates/interactions/bookmark_index.html:8]: do we need this `govuk-heading` and the class below? IMO when building new pages we should avoid any gds stuff

**Review by marcelkornblum — COMMENTED** [review]
I'm a bit confused about bookmark_page and bookmark_index - what are they both  there for and what's the difference?

A few other comments here and there but mostly 🤌

**Review by CamLamb — APPROVED** [review]

**Review by CamLamb — APPROVED** [review]

**Pull request merged**